extensions/blocks/membership-button/view.js

```diff
@@ -0,0 +1,57 @@
+/* global tb_show */
+
+/**
+ * Internal dependencies
+ */
+import './view.scss';
+const name = 'membership-button';
+const blockClassName = 'wp-block-jetpack-' + name;
+
+function activateSubscription( block, blogId, planId, poweredText ) {
+	block.addEventListener( 'click', () => {
+		tb_show(
```

Comment:
Could we use the [modal shipped in Gutenberg](https://github.com/WordPress/gutenberg/tree/6c9a9b22312acde7e64462f554fe3b9006becc9f/packages/components/src/modal)? Using these ancient jQuery libraries feels like we'll just introduce some serious maintenance dept.

Reply:
This modal is used on site frontend, not the administration part.
I opted out for not including the entirety of React library for every site visit.

Reply:
I agree with the sentiment — did you check if enqueuing entirety of jQuery, Dashicons and Thickbox is much different tho? 🙃